pyodide-build/pyodide_build/buildall.py

```diff
@@ -522,11 +522,6 @@ def generate_packagedata(
             }
             packages[name.lower() + "-tests"] = pkg_entry
 
-    # Workaround for circular dependency between soupsieve and beautifulsoup4
-    # TODO: FIXME!!
-    if "soupsieve" in packages:
-        packages["soupsieve"]["depends"].append("beautifulsoup4")
```

Comment:
I removed this because circular dependency is causing trouble and soupsieve docs mention that one needs to install beautifulsoup4 manually.(https://github.com/facelessuser/soupsieve/commit/9a792216ea1ffce018c7968105540c08b82b96db)